cypress/elements/dimensionModal/dataDimension.js

```diff
@@ -21,13 +22,13 @@ export const removeAllDataItems = () => cy.getBySel(removeAllButtonEl).click()
 export const expectNoDataItemsToBeSelected = () =>
     cy
         .getBySel(selectedItemEl)
-        .should('not.be.visible')
+        .should('be.visible')
         .and('have.length', 0)
 
 export const expectDataItemsAmountToBeSelected = amount =>
     cy
         .getBySel(selectedItemEl)
-        .should('not.be.visible')
+        .should('be.visible')
```

Comment:
technically out of scope, but this fn is unused and I noticed a typo in the assertion that I thought I'd fix while at it